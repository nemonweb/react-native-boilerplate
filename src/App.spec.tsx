// import React from 'react';

import { shallow } from 'enzyme'
import * as React from 'react'
import Hello from './App'

it('renders correctly', () => {
  const wrapper = shallow(<Hello name="World" enthusiasmLevel={1} />)
  expect(wrapper).toMatchSnapshot()
})
