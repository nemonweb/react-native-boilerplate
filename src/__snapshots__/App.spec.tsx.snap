// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "center",
    }
  }
>
  <Component
    style={
      Object {
        "color": "#999",
        "fontWeight": "bold",
      }
    }
  >
    Hello
     
    World!
  </Component>
  <Component
    style={
      Object {
        "alignItems": "stretch",
        "alignSelf": "center",
        "borderWidth": 5,
        "flexDirection": "row",
        "minHeight": 70,
      }
    }
  >
    <Component
      style={
        Object {
          "flex": 1,
          "paddingVertical": 0,
        }
      }
    >
      <Button
        accessibilityLabel="decrement"
        color="red"
        onPress={[Function]}
        title="-"
      />
    </Component>
    <Component
      style={
        Object {
          "flex": 1,
          "paddingVertical": 0,
        }
      }
    >
      <Button
        accessibilityLabel="increment"
        color="blue"
        onPress={[Function]}
        title="+"
      />
    </Component>
  </Component>
</Component>
`;
